src/core/CCFormat.mli

```diff
@@ -34,6 +34,15 @@ val exn : exn printer
 (** Printer using {!Printexc.to_string}.
     @since 3.0 *)
 
+val space : unit printer
```

Comment:
needs a docstring `@since NEXT_RELEASE` :-)

Reply:
Added in 94aa3cc